libbpf-tools/tcptop.bpf.c

```diff
@@ -12,9 +12,12 @@
 #define AF_INET		2	/* Internet IP Protocol 	*/
 #define AF_INET6	10	/* IP version 6			*/
 
-const volatile bool filter_cg = false;
-const volatile pid_t target_pid = -1;
-const volatile int target_family = -1;
```

Comment:
The above coding pattern has been used in most of libbpf-tools bpf programs. I would rather to keep this. Changing to use map does not sound good compared to more elegant way in the above. 4.19 is quite old. Maybe you could folk the repo and make change in your private repo?